### PR TITLE
replay: onboarding specific per sdk

### PR DESCRIPTION
**Pull request by bruno-garcia**
Resolves: https://github.com/getsentry/sentry-docs/issues/7045

### CODE REVIEW

File: src/platform-includes/session-replay/install/javascript.angular.mdx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+```bash {tabTitle:npm}
+# Angular 12 and newer:
+npm install --save @sentry/angular-ivy
```
Comment (ryan953) [inline src/platform-includes/session-replay/install/javascript.angular.mdx:3]: is this a new sdk name? we might need to include it in some lists in the app too

File: src/platform-includes/session-replay/install/javascript.angular.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+| Angular version | Recommended Sentry SDK |
+| --------------- | ---------------------- |
+| 12 and newer    | `@sentry/angular-ivy`  |
+| 10, 11          | `@sentry/angular`      |
+| Older versions  | See note below         |
+
+<Alert level="warning">
+
+Support for any Angular version below 10 was discontinued in version 7 of the SDK. If you need to use Angular 9 or older, try version 6 (`@sentry/angular@^6.x`) of the SDK. For AngularJS/1.x, use `@sentry/browser@^6.x` and our [AngularJS integration](/platforms/javascript/guides/angular/angular1). Note, that these versions of the SDK are no longer maintained or tested.
```
Comment (ryan953) [inline src/platform-includes/session-replay/install/javascript.angular.mdx:29]: replay only works with v7+

File: src/platform-includes/session-replay/install/javascript.angular.mdx (hunk 1 above)
Comment (bruno-garcia) [reply src/platform-includes/session-replay/install/javascript.angular.mdx:3]: Yeah, it's nuget trends uses too: https://github.com/dotnet/nuget-trends/blob/3b12f722505aca5423e469449549e46ab4986f6c/src/NuGetTrends.Web/Portal/package.json#L25-L26

Where do we need to include it?

Comment (ryan953) [reply src/platform-includes/session-replay/install/javascript.angular.mdx:3]: https://github.com/getsentry/sentry/blob/eee482eb315cc0bdae53dd0bddc96c95e7dc2f27/static/app/data/platformCategories.tsx#L330-L372 for sure.

I looked at some related onboarding/docs stuff... we might need to also create `angular-ivy-replay-onboarding-1-install.md` in the docs site too. I think it's meant to be moved into sentry repo, but i can't find the code here right now.

Comment (bruno-garcia) [reply src/platform-includes/session-replay/install/javascript.angular.mdx:3]: Oh that, It's already pointing to the right package: 
image

File: src/platform-includes/session-replay/install/javascript.capacitor.mdx [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
-yarn add @sentry/capacitor @sentry/replay
+Install the Sentry Capacitor SDK alongside the corresponding Sentry SDK for the framework you're using, such as Angular in this example:
+
+```bash {tabTitle:Angular}
+# npm
+npm install --save @sentry/capacitor @sentry/angular-ivy @sentry/replay
+
+# yarn
+yarn add @sentry/capacitor @sentry/angular-ivy @sentry/replay
 ```
```
Comment (ryan953) [inline src/platform-includes/session-replay/install/javascript.capacitor.mdx:9]: does the user need to install `@sentry/replay` if they're already doing `@sentry/angular-ivy` or `@sentry/vue` or whatever the framework is?

File: src/platform-includes/session-replay/install/javascript.capacitor.mdx [hunk 4]
```diff
@@ -1,7 +1,17 @@
...
 
-```html {tabTitle: npm}
+```bash {tabTitle:Other Frameworks}
+# npm
 npm install --save @sentry/capacitor @sentry/replay
+
+# yarn
+yarn add @sentry/capacitor @sentry/replay
 ```
```
Comment (ryan953) [inline src/platform-includes/session-replay/install/javascript.capacitor.mdx:17]: similar to above, should this be `@sentry/capacitor @sentry/javascript`?

File: src/platform-includes/session-replay/setup/javascript.angular.mdx
Comment (ryan953) [inline src/platform-includes/session-replay/setup/javascript.angular.mdx:1]: this file is copy+pasted a few times (9 times). I didn't notice any errors with that, but idk if we can or should de-dupe it to make maintenance easier. 


note that nextjs and remix and sveltekit have little headers at the top of the files which are nice to keep: "On your client-side NextJS app, add:"

**Review by ryan953 — APPROVED** [review]

File: src/platform-includes/session-replay/install/javascript.react.mdx [hunk 5]
```diff
@@ -0,0 +1,7 @@
+```bash {tabTitle:npm}
```
Comment (shanamatthews) [inline src/platform-includes/session-replay/install/javascript.react.mdx:1]:
```suggestion
The Replay integration is **already included** with the React SDK package.

```bash {tabTitle:npm}
```

File: src/platform-includes/session-replay/install/javascript.remix.mdx [hunk 6]
```diff
@@ -0,0 +1,23 @@
+We recommend installing the SDK through our installation wizard:
```
Comment (shanamatthews) [inline src/platform-includes/session-replay/install/javascript.remix.mdx:1]:
```suggestion
The Replay integration is already included with the Sentry SDK. We recommend installing the SDK through our installation wizard:
```

File: src/platform-includes/session-replay/install/javascript.svelte.mdx [hunk 7]
```diff
@@ -0,0 +1,7 @@
+```bash {tabTitle:npm}
```
Comment (shanamatthews) [inline src/platform-includes/session-replay/install/javascript.svelte.mdx:1]:
```suggestion
The Replay integration is **already included** with the Svelte SDK package.

```bash {tabTitle:npm}
```

File: src/platform-includes/session-replay/install/javascript.sveltekit.mdx [hunk 8]
```diff
@@ -0,0 +1,22 @@
+We recommend installing the SDK by running our installation wizard in the root directory of your project:
```
Comment (shanamatthews) [inline src/platform-includes/session-replay/install/javascript.sveltekit.mdx:1]:
```suggestion
The Replay integration is already included with the Sentry SDK. We recommend installing the SDK by running our installation wizard in the root directory of your project:
```

File: src/platform-includes/session-replay/install/javascript.vue.mdx [hunk 9]
```diff
@@ -0,0 +1,7 @@
+```bash {tabTitle:npm}
```
Comment (shanamatthews) [inline src/platform-includes/session-replay/install/javascript.vue.mdx:1]:
```suggestion
The Replay integration is **already included** with the Vue SDK package.

```bash {tabTitle:npm}
```

**Review by shanamatthews — APPROVED** [review]

File: src/platform-includes/session-replay/setup/javascript.angular.mdx [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+
+  // This sets the sample rate to be 10%. You may want this to be 100% while
+  // in development and sample at a lower rate in production
+  replaysSessionSampleRate: 0.1,
+
+  // If the entire session is not sampled, use the below sample rate to sample
+  // sessions when an error occurs.
```
[the remark this answers is not in the document]
Comment (bruno-garcia) [reply src/platform-includes/session-replay/setup/javascript.angular.mdx:12]: Sorry i don't see how this phrase makes it sound like it's 1.0 (also the actual value is right above it so unlikely confusing)?

That said: happy to change these comments to something that makes more sense. Due to all the copy paste we'd need a find/replace. Best as a follow up PR

File: src/platform-includes/session-replay/install/javascript.capacitor.mdx (hunk 4 above)
Comment (bruno-garcia) [reply src/platform-includes/session-replay/install/javascript.capacitor.mdx:17]: capacitor isn't part of the monorepo so I'm not sure if the re-export done works, but possibly so. I'll check and update on a follow up PR

Comment (bruno-garcia) [reply src/platform-includes/session-replay/install/javascript.capacitor.mdx:17]: cc @lucas-zimerman ^

File: src/platform-includes/session-replay/install/javascript.capacitor.mdx (hunk 3 above)
Comment (bruno-garcia) [reply src/platform-includes/session-replay/install/javascript.capacitor.mdx:9]: cc @lucas-zimerman ^

**Pull request merged**